addons/mail/models/mail_activity_plan.py

```diff
@@ -33,7 +33,7 @@ def _get_model_selection(self):
         help='Specify a model if the activity should be specific to a model'
               ' and not available when managing activities for other models.')
     steps_count = fields.Integer(compute='_compute_steps_count')
-    assignation_summary = fields.Html('Assignation summary', compute='_compute_assignation')
+    summary = fields.Html('Plan Summary', compute='_compute_summary')
```

Comment:
I wouldn't rename fields in stable... better keep the new naming, unless you have strong reasons to change it ?

Reply:
yes, thanks (actually, as discussed, I thought we were doing that before the merge)